models/linkages/R/write.restart.LINKAGES.R

```diff
@@ -77,10 +70,18 @@ write.restart.LINKAGES <- function(out.dir, runid, time, settings, analysis.vec,
   }
   
   spp.params <- spp.params.default[spp.params.save,]
+  
+  biomass_function<-function(dbh){ #kg/tree
+    .1193 * dbh^2.393 + ((slta+sltb*dbh)/2)^2 * 3.14 * fwt * frt * .001
+  }
+  merit<-function(dbh){
+    (b_obs - biomass_function(dbh))^2
```

Comment:
FWIW, I don't think the  `.` at the start forces the function to be internal, it's just a clue to developers that it is. I can't even find the reference I got that from, but I think it's probably related to discussions [here] (http://stackoverflow.com/questions/7526467/what-does-the-dot-mean-in-r-personal-preference-naming-convention-or-more), which note that (1) *variables* named with a leading `.` aren't shown by `ls()` by default, and (2) normal people wouldn't normally start typing a function name with a `.`. 

I think you should still stick to using roxygen tags to decide whether a function gets exported or not, but I think it's a fine convention (whether or not I pulled it out of my ass) to give non-exported functions names that start with `.` so it's readily apparent to other developers.